src/frontend/src/apis/transformers/change-requests.transformers.ts

```diff
@@ -12,15 +12,18 @@ import { ChangeRequest, ChangeRequestType, ImplementedChange } from 'shared';
  * @returns Properly transformed change request object.
  */
 export const changeRequestTransformer = (changeRequest: ChangeRequest) => {
-  const data: any = {
+  interface MaybeActiveChangeRequest extends ChangeRequest {
+    startDate?: Date;
+  }
+  const data: MaybeActiveChangeRequest = {
     ...changeRequest,
     implementedChanges: changeRequest.implementedChanges?.map(implementedChangeTransformer),
     dateSubmitted: new Date(changeRequest.dateSubmitted),
     dateReviewed: changeRequest.dateReviewed ? new Date(changeRequest.dateReviewed) : changeRequest.dateReviewed,
     dateImplemented: changeRequest.dateImplemented ? new Date(changeRequest.dateImplemented) : changeRequest.dateImplemented
   };
   if (changeRequest.type === ChangeRequestType.Activation) {
-    data.startDate = new Date(data.startDate);
+    if (data.startDate) data.startDate = new Date(data.startDate);
```

Comment:
Now I'm thinking that checking if data.startDate exists verifies if changeRequest.type is a type of Activation (since startDate property only exists in Activation change request). Therefore, I think we can remove the if statement on line 25 and replace it with this if statement on line 26.